### PR TITLE
Fixed coding styles

### PR DESCRIPTION
**Pull request by devenbansod**
@rajaths589 : This is how it looks when we use the standards I proposed on the chat.
Please review and let me know if you have any suggestions.

Signed-off-by: Deven Bansod devenbansod.bits@gmail.com

### CODE REVIEW

File: buffer.h [hunk 1]
```diff
@@ -5,71 +5,73 @@
...
-	const char diskpath = "/sys/block/";
-	char diskblkfile[] = (char*)malloc(sizeof(char)*(strlen(infofilepath)+strlen(devicename)+strlen(diskpath)));
-	strcat(diskblkfile,diskpath);
-	strcat(diskblkfile,devicename);
-	strcat(diskblkfile,infofilepath);
-	FILE* fptr = fopen(diskblkfile,"r");
-	size_t blksize;
-	fscanf(fptr,"%lu",&blksize);
-	return blksize;
+	const char info_file_path = "/queue/physical_block_size";
```
Comment (rajathshashidhara) [inline buffer.h:11]: I think this will be a compilation error.
const char info_file_path[] ?

**Comment by rajathshashidhara** [comment]
Don't commit these changes. I am working on a refined version of this stuff. Please wait for it.

**Comment by devenbansod** [comment]
Okay. Let me know. I will try to rebase my changes then. 
And anyways, I can not merge this (don't have the permissions). 

Should we shift these developments to the main repo's some other branch rather than yours ?

**Comment by rajathshashidhara** [comment]
I was thinking mihir will send me pull requests and I will send pull requests to main repo on behalf of team 1.

**Comment by devenbansod** [comment]
That seems fine too.

**Pull request closed without merging**